README.md

```diff
@@ -28,11 +28,17 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
   - [Want to hack on IPFS?](#want-to-hack-on-ipfs)
 - [License](#license)
 
-## Use Cases
+## Platforms
```

Comment:
That sounds right to me. Perhaps I am vague in my definition of `platforms`. Aren't these all apps, anyway?